charts/incubator/openbooks/values.yaml

```diff
@@ -0,0 +1,38 @@
+image:
+  repository: evanbuss/openbooks
+  pullPolicy: IfNotPresent
+  tag: 4.5.0@sha256:3f2e59a75b68950d57cfec5dc2b1b31d6c6b02706097c70b5438499b59fbdea2
+
+extraArgs:
+  - --port
+  - "{{ .Values.service.main.ports.main.port }}"
+  - --persist
+  - --name
+  - "{{.Values.env.USER_NAME}}"
+
+securityContext:
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+
+podSecurityContext:
+  runAsUser: 568
+  runAsGroup: 568
+
+env:
+  BASE_PATH: "/"
```

Comment:
If that's the default value, let's remove it.
Subfolder is not something we recommend or support

Reply:
i manually defined it as `/` since i read the docs that state the docker version says it makes it default to `/openbooks/`, i guess i was mistaken

Reply:
I don't know what the default is :P I just asked IF it is the default. :P

Reply:
I just checked the docs again i cant find it, it was just defined :P